commands/audit/jasrunner.go

```diff
@@ -30,6 +30,8 @@ func runJasScannersAndSetResults(scanResults *utils.Results, directDependencies
 	if progress != nil {
 		progress.SetHeadlineMsg("Running applicability scanning")
 	}
+	// Set environments variables for analytics in analyzers manager.
+	defer jas.SetAnalyticsMetricsDataForAnalyzerManager(msi, scanResults.GetScaScannedTechnologies())
```

Comment:
defer will only set the value after `runJasScannersAndSetResults` is done, after running the Applicability scan...